### DISCUSSION TITLE
Script wants to download everything no matter what I choose

### DISCUSSION THREAD

**Discussion by dummythicc99** [UltimaHoarder/UltimaScraper#1532] [Q&A]
Topic, Also, It seems to start scraping before I can even select which model I want, at least always creates new folders for every one of my subscriptions every time I run it.

So yea, it doesn't matter which sub I choose, or whether it's pics or vids, as soon as I hit enter it starts scraping EVERYTHING. Any help would be appreciated!

**Comment by LunarPenguin** [comment]
Could you provide us with a log or a screenshot?

**Reply by mayhemkrew** [reply]
I am seeing the same issue.  I have about 4 models I subscribe to and when I want to scrape just one, I select their number in the script and everything downloads.  @LunarPenguin what specifically are you looking for with logs?  I am new to this script and not sure where to look...

**Reply by LunarPenguin** [reply]
Im looking for the output of the script from the the start of the script until an error occurred or something happend that should not happen.
I hope that makes it clear enough.

**Reply by mayhemkrew** [reply]
> Im looking for the output of the script from the the start of the script until an error occurred or something happend that should not happen.
> I hope that makes it clear enough.

Well I have attached a gif of the script running.  You will see I tell it to select #4 model, then #2 for only video downloads.  It looks like it processes only that, but then at the end it starts grabbing all the content from all the models.  I cut it short mid-way through the 2nd model.  Hope that helps...

https://imgur.com/a/EoTBKvM

**Reply by LunarPenguin** [reply]
Thanks for the gif.

I myself have also seen this happen, somehow this is how the script works and im not sure why. i personally dont mind it because i want to almost always scrape all models anyways. but i agree that it is not how the program should work.

Maybe @DIGITALCRIMINALS can explain why this is happening and perhaps fix it.

**Reply by dummythicc99** [reply]
Sorry for the delayed response. I haven't ran the script in a while but what @mayhemkrew has posted pretty much sums up my issue.

I should clarify that while I originally thought it was grabbing everything else from other models, that wasn't the case. Let's say I want all of model B's content. The script will grab it all, but it will also grab a few images from model A and C. What's interesting I that it's the same exact images and videos from models A and C every time.
